src/coreclr/src/vm/eventtrace.cpp

```diff
@@ -5035,6 +5035,108 @@ VOID ETW::InfoLog::RuntimeInformation(INT32 type)
     } EX_CATCH { } EX_END_CATCH(SwallowAllExceptions);
 }
 
+/****************************************************************************/
+/* This is called by the runtime during startup to log the
+   properties passed on from the host. */
+/****************************************************************************/
+
+VOID ETW::InfoLog::RuntimeInputProperties(int nProperties, LPCWSTR* propertyNames, LPCWSTR* propertyValues)
+{
+    CONTRACTL{
+        NOTHROW;
+        GC_TRIGGERS;
+    } CONTRACTL_END;
+
+    EX_TRY{
+        if (ETW_EVENT_ENABLED(MICROSOFT_WINDOWS_DOTNETRUNTIME_PROVIDER_DOTNET_Context, RuntimeProperties))
+        {
+            unsigned short ClrInstanceID = GetClrInstanceId();
+            LPCWSTR trustedPlatformAssemblies = nullptr;
+            LPCWSTR nativeDllSearchDirectories = nullptr;
+            LPCWSTR platformResourceRoots = nullptr;
+            LPCWSTR appContextBaseDirectory = nullptr;
+            LPCWSTR appContextDepsFiles = nullptr;
+            LPCWSTR probingDirectories = nullptr;
+            LPCWSTR fxProductVersion = nullptr;
+            LPCWSTR jitPath = nullptr;
+            LPCWSTR fxDepsFile = nullptr;
+            LPCWSTR appPaths = nullptr;
```

Comment:
`appPaths` and `appNIPaths` are deprecated and not used anywhere. Do we really need to log them?

I would rather deleted them.